rcswitch/user_doc.rst

```diff
@@ -0,0 +1,134 @@
+.. index:: Plugins; rcswitch (Brennenstuhl RCS 1000 N)
+.. index:: rcswitch
+
+================
+Plugins/rcswitch
+================
+
```

Comment:
Unter dem Titel (Plugin Name) vor dem ersten Text-Abschnitt der Dokumentation sollte das Plugin Logo dargestellt werden.

Das erfolgt normalerweise durch einfügen der folgenden Zeilen (.. image: Tag):
```
=========
rcswitch2
=========

.. image:: webif/static/img/plugin_logo.png
   :alt: plugin logo
   :width: 300px
   :height: 300px
   :scale: 50 %
   :align: left
```

Reply:
erl. in PR für `rcs1000n`